static/js/actions/course.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+import { createAction } from "redux-actions"
+
+export const SET_COURSE_DATA = "SET_COURSE_DATA"
+export const setCourseData = createAction(SET_COURSE_DATA)
```

Comment:
I see this action defined here, and a handler defined in the reducer, but it doesn't look like it's called anywhere

Reply:
Removed